tests/conftest.py

```diff
@@ -501,6 +501,24 @@ async def trained_response_selector_bot(trained_async: Callable) -> Path:
     return Path(zipped_model)
 
 
+@pytest.fixture(scope="session")
+async def trained_formbot(trained_async: Callable) -> Path:
```

Comment:
Why `formbot`?

Reply:
Why I named it formbot? Because I took the least data possible from `examples/formbot`.

Reply:
I was referring to the name. There is no form in the bot now right? So maybe a different name would be clearer

Reply:
Great catch! 💯 I didn't even think about it. I just took it from the formbot so I kept the same name and when I removed the basic logic that makes this bot a formbot, I did not think of renaming it. It is so simple but so easy to miss it.